cmd/bridgec/erc20cmd.go

```diff
@@ -0,0 +1,111 @@
+	erc20SubCmds := &erc20SubCmds{cli: client}
+
+	// define Rootcommand
+	var (
+		rootCmd = &cobra.Command{
+			Use:   "erc20",
+			Short: "Perform erc20 actions",
+			Long:  "Perform erc20 actions",
+			Run:   erc20SubCmds.getSyncingStatus,
+		}
+		getSyncingStatusCmd = &cobra.Command{
+			Use:   "syncstatus",
+			Short: "Get the ethereum sync status",
+			Long:  `Get the ethereum chain sync status.`,
+			Run:   erc20SubCmds.getSyncingStatus,
+		}
+		getBalanceInfoCmd = &cobra.Command{
+			Use:   "balance",
+			Short: "Get the ethereum balance info",
+			Long:  `Get the ethereum balance and address information`,
+			Run:   erc20SubCmds.getBalanceInfo,
+		}
+	)
+
+	rootCmd.AddCommand(getSyncingStatusCmd)
+	rootCmd.AddCommand(getBalanceInfoCmd)
+
+	// register flags
+	getSyncingStatusCmd.Flags().Var(
+		cli.NewEncodingTypeFlag(cli.EncodingTypeHuman, &erc20SubCmds.getSyncingStatusCfg.EncodingType, cli.EncodingTypeHuman|cli.EncodingTypeJSON), "encoding",
+		cli.EncodingTypeFlagDescription(cli.EncodingTypeHuman|cli.EncodingTypeJSON))
+	getBalanceInfoCmd.Flags().Var(
+		cli.NewEncodingTypeFlag(cli.EncodingTypeHuman, &erc20SubCmds.getBalanceInfoCfg.EncodingType, cli.EncodingTypeHuman|cli.EncodingTypeJSON), "encoding",
+		cli.EncodingTypeFlagDescription(cli.EncodingTypeHuman|cli.EncodingTypeJSON))
+
+	return rootCmd
+}
+
+type erc20SubCmds struct {
+	cli                 *internal.CommandLineClient
+	getSyncingStatusCfg struct {
+		EncodingType cli.EncodingType
+	}
+	getBalanceInfoCfg struct {
+		EncodingType cli.EncodingType
+	}
+}
+
+// getSyncingStatus Gets the ethereum blockchain syncing status from the deamon API
+func (erc20SubCmds *erc20SubCmds) getSyncingStatus(cmd *cobra.Command, args []string) {
+	var syncingStatus api.ERC20SyncingStatus
+
+	err := erc20SubCmds.cli.GetAPI("/erc20/downloader/status", &syncingStatus)
+	if err != nil {
+		cli.DieWithError("error while fetching the syncing status", err)
+	}
+
+	// encode depending on the encoding flag
+	switch erc20SubCmds.getSyncingStatusCfg.EncodingType {
+	case cli.EncodingTypeHuman:
+		fmt.Printf(`Starting block height: %d
+Current block height: %d
+Highest block height: %d
+`, syncingStatus.Status.StartingBlock, syncingStatus.Status.CurrentBlock, syncingStatus.Status.HighestBlock)
+	case cli.EncodingTypeJSON:
+		err = json.NewEncoder(os.Stdout).Encode(syncingStatus.Status)
+		if err != nil {
+			cli.DieWithError("failed to encode syncing status", err)
+		}
+	}
+}
+
+// getSyncingStatus Gets the ethereum blockchain syncing status from the deamon API
+func (erc20SubCmds *erc20SubCmds) getBalanceInfo(cmd *cobra.Command, args []string) {
+	var balanceInfo api.ERC20BalanceInformation
+
+	err := erc20SubCmds.cli.GetAPI("/erc20/account/balance", &balanceInfo)
+	if err != nil {
+		cli.DieWithError("error while fetching the balance information", err)
+	}
+
+	// encode depending on the encoding flag
+	switch erc20SubCmds.getSyncingStatusCfg.EncodingType {
+	case cli.EncodingTypeHuman:
+		fmt.Printf(`
```

Comment:
```suggestion
		fmt.Printf(`Address: %s
Balance: %d
```

Start the Address on the first line, do not keep the first line empty. Not that I mind, but following the convention of this file, this is the only place where you only start the content on the 2nd line.